### ISSUE TITLE
[CoE Starter Kit - QUESTION] Core Components 3.39 Connector Import Failure 0x80060439

### ISSUE THREAD

**Issue by matthewdevaney** [microsoft/coe-starter-kit#3059]
### What is your question?

When I attempt to import the Center of Excellence Core Components 3.39 managed into my environment I get the following error message "Connector import: FAILURE: One or more errors occurred."

![image](https://user-images.githubusercontent.com/34290969/175813743-bbb2320b-8ac4-478b-8b84-6ea45ebd977b.png)

I checked the error logs and it stopped processing at this customization.

admin_PowerPlatformAdminView

![image](https://user-images.githubusercontent.com/34290969/175823596-6993d791-ff43-4c9e-8e50-ef95058728e4.png)

This Center Of Excellence version installed successfully a few days before.  I deleted it from the environment so I could start again and document the process.  After my failed re-attempt I deleted the environment and created a new environment with a different name and had no luck.

What does the error message mean and how can I fix it?

### What solution are you experiencing the issue with?

CenterofExcellenceCoreComponents_3.39_managed.zip

### What solution version are you using?

3.39

### What app or flow are you having the issue with?

admin_PowerPlatformAdminView

**Comment by Jenefer-Monroe** [comment]
Can you go to the solution history and get the "More Details"? 
![image](https://user-images.githubusercontent.com/89584709/175983014-c8cf3240-d9d5-4326-8512-297e93a47b26.png)

**Comment by matthewdevaney** [comment]
@Jenefer-Monroe 
Here is the "More Details" screenshot you requested.

![image](https://user-images.githubusercontent.com/34290969/175989442-01657b58-50a1-4784-9dcc-c3c58fbd9f18.png)

**Comment by Jenefer-Monroe** [comment]
Well that's not very helpful unfortunately. 
As far as I can tell you have hit an intermittent product issue, since this worked before and isnt working now, and we havent heard anyone else repro this. 

Try importing without setting any env variables, and try installing via the old UX perhaps. 

![image](https://user-images.githubusercontent.com/89584709/175995190-23563b4a-d222-47a2-aac1-744d136b7b6c.png)

![image](https://user-images.githubusercontent.com/89584709/175995274-c3cf8731-cd8c-4f24-9383-bc06ed3d5cce.png)

**Comment by matthewdevaney** [comment]
@Jenefer-Monroe 
I will try both of your suggestions right now and report back soon.

**Comment by matthewdevaney** [comment]
@Jenefer-Monroe 
I tried both of your suggestions to use modern import without enviroment variables and the classic import.  Unfortunately neither worked.

![image](https://user-images.githubusercontent.com/34290969/176004491-ccb27aae-055a-40da-a760-353f21c58fd6.png)

I got a different error message when importing in classic.

"An error has occurred. Try this action again. If the problem continues, check the Microsoft Dynamics 365 Community for solutions or contact your organization's Microsoft Dynamics 365 Administrator. Finally, you can contact Microsoft Support. : System.AggregateException: One or more errors occurred. ---> Microsoft.Crm.Metadata.AttributeMetadataNotFoundException: 'connector' entity doesn't contain attribute with Name = 'customcodeblobcontent' and NameMapping = 'Platform'. MetadataCacheDetails: ProviderType=Dynamic, StandardCache=False, IsLoadedInStagedContext = True, Timestamp=1021668, MinActiveRowVersion=1021668, MetadataInstanceId=8063748, LastUpdated=2022-06-26 11:40:27.540, OrgId=60be2d56-6b46-4a98-8406-aa1cef2edc37    at Microsoft.Crm.Metadata.EntityMetadata.GetAttribute(String attributeName, NameMappingType mapping)    at Microsoft.Crm.BusinessEntities.BusinessEntity.GetAttributeMetadata(String attributeName)    at Microsoft.Crm.BusinessEntities.BusinessEntity.ValidateForRead(String attributeName)    at Microsoft.Crm.BusinessEntities.BusinessEntity.get_Item(String attributeName)    at Microsoft.Crm.ObjectModel.ConnectorService.<UpdateConnectorCustomCode>d__62.MoveNext()    --- End of inner exception stack trace ---    at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)    at Microsoft.Crm.ObjectModel.ConnectorService.SaveBackingConnectorComponentsIfNecessary(ExecutionContext context, IBusinessEntity connector, String existingConnectorInternalId)    at System.Threading.Tasks.Task`1.InnerInvoke()    at System.Threading.Tasks.Task.Execute() --- End of stack trace from previous location where exception was thrown ---    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)    at Microsoft.Crm.ObjectModel.ConnectorService.Create(IBusinessEntity entity, ExecutionContext context)    at Microsoft.Crm.Tools.ImportExportPublish.ImportEntityHandlerBase.<>c__DisplayClass4_0.<EnsureEntityExisted>b__0()    at Microsoft.PowerApps.CoreFramework.ActivityLoggerExtensions.Execute(ILogger logger, EventId eventId, ActivityType activityType, Action action, IEnumerable`1 additionalCustomProperties)    at Microsoft.Xrm.Telemetry.XrmTelemetryExtensions.Execute(ILogger logger, XrmTelemetryActivityType activityType, Action action)    at Microsoft.Crm.Tools.ImportExportPublish.ImportConnectorHandler.ImportItem() ---> (Inner Exception #0) Microsoft.Crm.Metadata.AttributeMetadataNotFoundException: 'connector' entity doesn't contain attribute with Name = 'customcodeblobcontent' and NameMapping = 'Platform'. MetadataCacheDetails: ProviderType=Dynamic, StandardCache=False, IsLoadedInStagedContext = True, Timestamp=1021668, MinActiveRowVersion=1021668, MetadataInstanceId=8063748, LastUpdated=2022-06-26 11:40:27.540, OrgId=60be2d56-6b46-4a98-8406-aa1cef2edc37    at Microsoft.Crm.Metadata.EntityMetadata.GetAttribute(String attributeName, NameMappingType mapping)    at Microsoft.Crm.BusinessEntities.BusinessEntity.GetAttributeMetadata(String attributeName)    at Microsoft.Crm.BusinessEntities.BusinessEntity.ValidateForRead(String attributeName)    at Microsoft.Crm.BusinessEntities.BusinessEntity.get_Item(String attributeName)    at Microsoft.Crm.ObjectModel.ConnectorService.<UpdateConnectorCustomCode>d__62.MoveNext()<--- "

**Comment by Jenefer-Monroe** [comment]
No I'm afraid not. This appears to be a product issue specific to your tenant configuration. Here is an early take at the the July release. You can try that perhaps. 
[CenterofExcellenceCoreComponents_3.40_managed.zip](https://github.com/microsoft/coe-starter-kit/files/9015349/CenterofExcellenceCoreComponents_3.40_managed.zip)

But this does appear to be a product issue and seems specific to something about your tenant configuration.

**Comment by matthewdevaney** [comment]
@Jenefer-Monroe 
This issue resolved itself a few days later.  The CoE Starter Kit successfully installed.  Its my own personal tenant and I did not make any changes to it.  I can only explain this one way.  Microsoft must have made changes to my environment/tenant behind the scenes.  I followed the exact same process for install and had not logged in since my last install attempt.

CASE CLOSED.  Thanks a bunch for your help!
